### DISCUSSION TITLE
Custom expiration email notification

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#417] [Feature Requests]
At the moment, the PublicPress Future notifies the specified email in the settings about an expired post on the day of such expiry by default. I would like the notification date feature to be customised, such that I can change the date of notification, e.g. to the day before. This is so that I can edit the expiry date if I so choose, in time before the expiry date

**Comment by andergmartins** [comment]
This is possible now using Action Workflows, with the Schedule and Send Email steps.
